src/Docs/CLI/Evaluate.hs

```diff
@@ -384,7 +384,7 @@ evaluate input =
     Left err   -> liftIO (putStrLn err)
     Right cmd  -> evaluateCmd cmd `catchError` showFailure
   where
-    showFailure e = liftIO $ putStrLn $ "Failed: "<> e
+    showFailure e = liftIO $ hPutStrLn stderr $ "Failed: "<> e
```

Comment:
I also swapped some of the status and error messaging to use stderr so that stdout is more easily parseable by scripts and such. It should look exactly the same in regular terminal usage.